modules/microshift-auto-apply-manifests.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * microshift/using-config-tools.adoc
+
+:_content-type: CONCEPT
+[id="microshift-auto-apply-manifests_{context}"]
+= Auto-applying manifests
+
+When you run `kustomize`, {product-title} uses the output and applies it to the cluster. At start-up, it searches `/etc/microshift/manifests` and `/usr/lib/microshift/` manifests directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k command` to apply that manifest. Providing multiple directories allows a flexible method of managing {product-name} workloads.
+
+[IMPORTANT]
+====
+`Kustomize` must be either running as a separate step in the boot process, or it must be part of the {product-title} image. 
+====
+
+[cols="2",options="header"]
+|===
+|Location
+|Intent
+
+|`/etc/microshift/manifests`
+|Read-write location for configuration management systems or development
+
+|`/usr/lib/microshift/manifests`
+|Read-only location for embedding configuration manifests on ostree based systems
+|===
+
+== Manifest Example
+This example demonstrates automatic deployment of a busybox container using `kustomize` manifests in the `/etc/microshift/manifests` directory.
+
+.Procedure
+Create the busybox manifest files by running the following commands:
+
+[source,terminal]
+----
+MANIFEST_DIR=/etc/microshift/manifests
+sudo mkdir -p ${MANIFEST_DIR}
+sudo tee ${MANIFEST_DIR}/busybox.yaml &>/dev/null <<EOF
+----
+
+.Busybox deployment YAML file example
```

Comment:
Splitting the YAML content from the command like this makes it render a bit odd. The command in the original doc included the YAML content, too, and looking over at that I'm not sure the syntax was actually correct. https://github.com/openshift/microshift/blob/main/docs/howto_config.md

Instead of the command on line 38 we could just say "create a file busybox.yaml containing" the content here on lines 44-67 (not including the "EOF" on line 68).

That can be fixed in another PR if we want to keep moving with this review.